### DISCUSSION TITLE
tab bar height

### DISCUSSION THREAD

**Discussion by lucasvs** [wezterm/wezterm#2519] [Q&A] [answered]
how to decrease the tab bar height?

**Comment by wez** [comment, marked as answer]
Change the font size; see https://wezfurlong.org/wezterm/config/appearance.html#native-fancy-tab-bar-appearance

**Reply by lucasvs** [reply]
Thank you it works, the line of the pane don't go to the end when I change the font size of the tab is this normal?
![image](https://user-images.githubusercontent.com/3601889/190239017-bf4e7ae2-e46f-4fbd-a581-b8f2d843c631.png)
